server/app/test_task.py

```diff
@@ -0,0 +1,244 @@
+from django.test import TestCase
+from django.utils import timezone
+from .models import User, Task
+from django.core.exceptions import ValidationError
+
+
+class TaskModelTests(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create(
+            email="testuser@example.com",
+            mobile="1234567890",
+            password_hash="password123",
+            status="active"
+        )
+        self.task = Task.objects.create(
+            owner_id=self.user,
+            title="Test Task",
+            category="General",
+            description="This is a test task",
+            location="Test Location",
+            estimated_price="100",
+            estimated_time="2 hours",
```

Comment:
Same as above: could the estimated_time field be numerical rather than a string?

Reply:
I just change it to 100